monkey-runner/plugin/build.gradle

```diff
@@ -0,0 +1,20 @@
+apply plugin: 'com.novoda.bintray-release'
+apply plugin: 'groovy'
+
+repositories {
+    jcenter()
+}
+
+dependencies {
+    compile gradleApi()
+    compile 'com.novoda:gradle-android-command-plugin:1.6.2'
```

Comment:
The other build.gradle is using `1.5.0`. Using common declared dependencies could help avoiding such issues.